GridTariffAPI.lib/Services/Helpers/ControllerValidationHelper.cs

```diff
@@ -28,6 +28,20 @@ public ControllerValidationHelper()
 
         }
 
+
+        public async Task<bool> ValidateTariffExistsAsync(TariffQueryRequest request)
```

Comment:
Foreslår å slette denne, for den brukes ikke lenger (annet enn fra en test)

Reply:
Implementert,